third_party/terraform/website/docs/r/compute_attached_disk.html.markdown

```diff
@@ -104,6 +104,12 @@ The following arguments are supported:
 	  "READ_ONLY"
 	  "READ_WRITE"
 
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource with format `projects/{{project}}/triggers/{{trigger_id}}`
```

Comment:
I don't think this example is for an attached disk.

Reply:
You are correct. The id value has been updated. Thank you